eslint-configs/eslint-config-seekingalpha-typescript/rules/typescript-eslint/index.js

```diff
@@ -0,0 +1,389 @@
+module.exports = {
+  plugins: ['@typescript-eslint/eslint-plugin'],
+
+  rules: {
+
+    'import/extensions': [
```

Comment:
I think you should drop it from here, this should be updated in `https://github.com/seekingalpha/javascript/tree/master/eslint-configs/eslint-config-seekingalpha-base`